edge/cmd/edgecore/app/server.go

```diff
@@ -54,6 +55,11 @@ is the message processor between edged and edgehub. It is also responsible for s
 to/from a lightweight database (SQLite).ServiceBus is a HTTP client to interact with HTTP servers (REST),
 offering HTTP client capabilities to components of cloud to reach HTTP servers running at edge. `,
 		Run: func(cmd *cobra.Command, args []string) {
+			// print version
+			if opts.ShowVersion {
+				fmt.Printf("version: %#v\n", version.Get())
+				return
+			}
```

Comment:
If we develop based on a certain release version, what does the returned version look like? For example, `v1.19.0-dirty` or still `v1.19.0`?

Reply:
This may depend on the implementation of the `version.Get()` function, which I call directly.